.github/workflows/weekly.yml

```diff
@@ -80,7 +80,7 @@ jobs:
       - name: Install dependencies
         run: pip install -r requirements-dev.txt
       - name: Installing Avocado in develop mode
-        run: python3 setup.py develop --user --skip-optional-plugins
+        run: python3 setup.py develop --user --skip-optional-plugins --skip-examples-tests-plugins
```

Comment:
I am not sure with this change, because this job runs all the selftests without optional plugins (see line 89). Which means that it runs `selftests/functional` tests which might use the `examples-tests-plugins`.  If we disable them then those tests might fail.

Reply:
That's a good point... We don't have a use case yet where a "core" self test will actively use an example plugin, but we'll soon have a couple (both your work and my work should require that).

AFAIK, this check is about having Avocado tested without any of the non-core plugins, so that would extend to the example plugins.  It should reflect the behavior you'd get with a `pip install avocado-framework` IMO.

This brings complications to the "core" self tests that may need to rely on the example plugins being active.  My suggestion is that we keep things like this, and on tests that require the example plugins, we check on their presence, example:

```python
@skipUnless(python_module_available("avocado-magic"))
```

This way we can keep the job with its original purpose (AFAIK), and still rely on plugins on tests.

Reply:
But, there's actually a typo in that line.  It should read ```--skip-examples-plugins-tests``` instead.

Reply:
Thanks for the explanation, It makes sense to me. I am a little afraid that in the future we will forget to add `@skipUnless` to new tests since it is not visible that those examples might be disabled. But IMO we can try it like this and fix those problems when they will occur.